internal/venice-client-common/src/main/java/com/linkedin/venice/exceptions/ErrorType.java

```diff
@@ -15,7 +15,7 @@ public enum ErrorType {
   SCHEMA_NOT_FOUND(ExceptionType.SCHEMA_NOT_FOUND), CONNECTION_ERROR(ExceptionType.CONNECTION_ERROR),
   @JsonEnumDefaultValue
   GENERAL_ERROR(ExceptionType.GENERAL_ERROR), BAD_REQUEST(ExceptionType.BAD_REQUEST),
-  CONCURRENT_BATCH_PUSH(ExceptionType.BAD_REQUEST);
+  CONCURRENT_BATCH_PUSH(ExceptionType.BAD_REQUEST), PRECONDITION_CHECK_FAILED(ExceptionType.BAD_REQUEST);
```

Comment:
I think this error should be more specific. The reason for this enum was to print readable error messages

Also, `BAD_REQUEST` implies user error. This seems to be a system error

Reply:
Yes. `RESOURCE_STILL_EXISTS` is perfect. Looking at the impl more, `BAD_REQUEST` is okay since we expect the cleanup to only happen on stores that have no state left over